src/app/views/rolodex_view.js

```diff
@@ -0,0 +1,83 @@
+import Backbone from 'backbone';
+import $ from 'jquery';
+import _ from 'underscore';
+import Contact from '../models/contact.js';
+import ContactView from '../views/contact_view.js';
+
+var RolodexView = Backbone.View.extend({
+  initialize: function(params) {
+    this.contactTemplate = params.template;
+    this.listenTo(this.model, "update", this.render);
+    // Wave3, compling modal template and adding it to html
+    this.modalTemplate = _.template($('#tmpl-contact-details').html());
+    console.log('this modalTemplate ', this.modalTemplate);
+  },
+
+  render: function(){
+    this.$('#contact-cards').empty();
+    var that = this;
+
+    this.model.each(function(contact){
+      var contactView = new ContactView({
+        model: contact,
+        myTemplate: that.contactTemplate,
+      });
+      // Wave3 create listener for each contact to be filled in the modal
+      that.listenTo(contactView, "selected", that.viewModal);
+
+      // Wave2, displaying list of contacts using rolodex collection
+      that.$("#contact-cards").append(contactView.render().$el);
+    });
+
+    return this;
+  },
+
+// Wave3 backbone modal event handler
+  viewModal: function(contact){
+    $('#contact-details').show();
```

Comment:
It's better to avoid using raw jQuery to select items, rather stick to using `this.$` to select elements within the view.  That prevents you from selecting something outside the view.